packages/pixeloven/cli-addon-storybook/src/commands/story.ts

```diff
@@ -1,32 +1,36 @@
-import { AddonStorybookRunContext, StorybookExecutionType } from "../types";
+import { AddonStorybookToolbox, StorybookExecutionOptionTypes, StorybookExecutionType } from "../types";
 
 export default {
     alias: ["--story", "-s"],
     name: "story",
-    run: async (context: AddonStorybookRunContext) => {
-        const { parameters, pixelOven, storybook } = context;
-        if (!parameters.first) {
-            pixelOven.invalidArgument("Must provide a story run type.");
+    run: async (toolbox: AddonStorybookToolbox) => {
+        const { parameters, pixelOven, storybook } = toolbox;
+        const task = parameters.first;
```

Comment:
Nit: parameters.first is already pretty clear and reduces the need for re-assigning variables.

Reply:
Ehh nvm... not worth changing since now we have consistency with webpack and storybook.

Reply:
I found it easier to grok thinking of the first parameter as a task and the others as the options to that task so I figured naming it as such might help others as well.